src/pages/product/Product.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { AiFillHeart } from "react-icons/ai";
+
+const Product = () => {
+
+    const [item, setItem] = useState([]);
```

Comment:
Variable name could be `[products,setProducts]`

Reply:
done